ofl/playpensans/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1,63 @@
+</p>
+<img src="PlaypenSans-about1a.png">
+<p>
+The font world has a general tension between what’s organic and what’s digital. When scribbling a quick note, the letters have slight differences, but all look similar because they come from the same person. Handwritten text is inconsistent, while digital typefaces are intended to be consistent — each character exactly the same every time it’s typed.
+</p>
+<p>    
+The goal of a typeface that is both casual in look and digital in nature is to appear authentically human within the bounds of digital reproduction. So, a typeface with a set of characters that are “the same but different” has the authenticity everyone craves. The main problem with casual fonts is not having enough alternate characters to look real. When a family has more than one alternate, another problem arises in controlling how and when a character gets replaced.
+</p>
+<img src="PlaypenSans-about2a.png">
+<p>    
+To solve these problems, we designed Playpen Sans with seven versions of each character. We also created a built-in shuffler so no single shape is repeated in close proximity. The result is text with spontaneous inconsistencies that feel fun and organic… all the benefits of a modern, professional typeface that looks natural.
+</p>
+<p>    
+Playpen Sans has eight weights emojis for breezy and encouraging uses and supports over 150 Latin-based languages. Its straight and curved endings for ‘i, l, y’, the two-story ‘a’, and optional shapes for ‘f, G, I, M’ are clarifying features. The family was made for non-designers and to shine within short, informal settings: greeting cards and invitations, casual signs and documents, and of course, children’s books and educational materials, comic books, and graphic novels. It also comes in a variable font version for developers and designers who value ultimate control, data savings, and design superiority.
+</p>
+
+<p>    
+Playpen Sans is the extension of technological and aesthetic potential, showing the best of both worlds with its digital capabilities and casual, handmade look. Is it spontaneous? Is it authentic? Thankfully, yes, and yes.
+</p>
+<p>
+To contribute, see <a href="https://github.com/TypeTogether/Playpen-Sans.git" target="_blank">github.com/TypeTogether/Playpen-Sans</a>
+</p>
+<img src="PlaypenSans-about3a.png">
+<hr>
+<table>
+    <tr>
+        <th><h3>Emoji you place</h3></th>
+        <th><h3>Emoji you get</h3></th>
+    </tr>
+    <tr>
+        <td>🌠</td>
+        <td><img src="e-shooting-star.png"></td>
+    </tr>
+    <tr>
+        <td>🎂</td>
+        <td><img src="e-birthday-cake.png"></td>
+    </tr>
+    <tr>
+        <td>🎨</td>
+        <td><img src="e-color-palette.png"></td>
+    </tr>
+    <tr>
+        <td style="font-size: 30px;">👆</td>
+        <td><img src="e-index-pointing.png"></td>
+    </tr>
+</table>
```

Comment:
the `table`, `tr` and `td` elements are not listed in the allowed elements for the article

Reply:
AFAIR Chris confirmed with Nate that tables were also supported.